src/app/personside/infotabs/oppfolging/OppfolgingDatoKomponent.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import Undertittel from 'nav-frontend-typografi/lib/undertittel';
+import theme from '../../../../styles/personOversiktTheme';
+import Datovelger from 'nav-datovelger/dist/datovelger/Datovelger';
+import { Knapp } from 'nav-frontend-knapper';
+
+const Wrapper = styled.div`
```

Comment:
vi prøver å unngå "wrapper" som navn på styled components. Noe mer beskrivende? Kan være så enkelt som "OppfolgingDetaljerStyling" for min del